users/service.go

```diff
@@ -302,6 +310,9 @@ func (svc usersService) ChangePassword(ctx context.Context, authToken, password,
 	if err != nil {
 		return errors.Wrap(ErrUnauthorizedAccess, err)
 	}
+	if !svc.passRegex.MatchString(password) {
```

Comment:
Should we validate password format in the `ResetPassword` metod?

Reply:
Definitely